src/main/java/com/jorgepatrick/RomanCalculator.java

```diff
@@ -0,0 +1,54 @@
+package com.jorgepatrick;
+import static com.jorgepatrick.RomanNumbers.*;
+
+public class RomanCalculator {
+    public String sumRomanNumbers(final String firstAddend, final String secondAddend) {
+
+        final int firstArabicNumber = parseRomanToArabic(firstAddend);
+        final int secondArabicNumber = parseRomanToArabic(secondAddend);
+
+        int arabicNumberSum = firstArabicNumber + secondArabicNumber;
+
+        return parseArabicToRoman(arabicNumberSum);
+    }
+
+    private String parseArabicToRoman(final int arabicNumberSum) {
+        String romanNumber = "";
+        int unitQuantity = 0;
+
+        if (arabicNumberSum == 5) {
+            return V;
+        }
+
+        if (arabicNumberSum == 4) {
+            return IV;
+        }
+
+        if (arabicNumberSum > 5) {
+            romanNumber = V;
+            unitQuantity = arabicNumberSum - 5;
+
+        } else {
+            unitQuantity = arabicNumberSum;
+        }
+
+        for (int i = 0; i < unitQuantity; i++) {
+            romanNumber += 'I';
```

Comment:
You can use the constant `I` here as well

Reply:
Havent used because itsnt exactly the roman number, it is the char 'I', should I,ve used the constant?